gtda/diagrams/features.py

```diff
@@ -154,14 +154,14 @@ class BettiCurve(BaseEstimator, TransformerMixin):
     See also
     --------
     PersistenceLandscape, PersistenceEntropy, HeatKernel, Amplitude, \
-    PairwiseDistance, gtda.homology.VietorisRipsPersistence
+    PairwiseDistance, Silhouette, gtda.homology.VietorisRipsPersistence
 
     Notes
     -----
     The samplings in :attr:`samplings_` are in general different between
     different homology dimensions. This means that the j-th entry of a Betti
     curve in homology dimension q typically arises from a different parameter
-    values to the j-th entry of a curve in dimension q'.
+    values to the j-th entry of a curve in dimension q-1'.
```

Comment:
I'm not sure I agree with this change. The point was to have `q` and `q'` generally different.

Reply:
For some reason, I did not see the prime. sorry